Guides/Python/HelloWorld.md

```diff
@@ -36,6 +37,31 @@ web: python server.py
 
 Left from the colon we specified the **required** process type called `web` followed by the command that starts the app and listens on the port specified by the environment variable `$PORT`.
 
+###The Actual Application Code
+
+The actual application code is really straight forward. We import the os module and  
+the Flask class and created an instance of the class. Next we took the route()
+decorator to tell Flask which URL should trigger our hello() function.
+When the URL is triggered, the render_template() function renders the hello.jinja
+template.
+Finally we use the run() function with the parameters host=0.0.0.0 and
+port with an instance of os.environ.
+
+~~~python
+import os
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello():
+    return render_template('hello.jinja', domain=os.environ['DOMAIN'])
+
+app.debug = True
+app.run(host='0.0.0.0', port=int(os.environ['PORT']))
+~~~
+
```

Comment:
Part of the reason why we do this is also to unify the different guides more. Nad we have this in a lot of other ones. Basically the idea is to provide everything in the repo ready to go, but still explain the most important files. Usually the Procfile, and the code that is started with the command in the Procfile.

I think including the code is good. We could shorten the how the code works explanation above though. I also feel this is a little too verbose for code this simple.